### PR TITLE
implement /api/v1/utxo?block=<height>

### PR DESCRIPTION
**Pull request by cleverca22**

### CODE REVIEW

**Comment by erikd** [comment]
* The "address review comments" commit should not be there.
* Warnings in `test/Test/IO/Explorer/Web/Query.hs` which should have failed the build with `-Werror`
* We have debug output mixed in with the test output.
* The new tests should fail with an nice error message if the setup is incorrect (the tests are currently failing which I think is due to incorrect DB setup).

File: cardano-explorer-db/cardano-explorer-db.cabal [hunk 1]
```diff
@@ -30,6 +30,7 @@ library
                         -Wincomplete-uni-patterns
 
   exposed-modules:      Explorer.DB
+                        Test.IO.Explorer.DB.Util
```
Comment (erikd) [inline cardano-explorer-db/cardano-explorer-db.cabal]: Still need to fix that as well.

**Comment by cleverca22** [comment]
1: i will squash the whole thing once the review is green
2: `{-# OPTIONS_GHC -Wno-error=orphans #-}`, the orphaned instances are needed by tests, and i thought id keep them visible
3: i could switch it to `runNoLoggingT` but maybe it should be a flag, so it can be turned easily, maybe just a `logMode = runNoLoggingT` at the top-level, since its not going to be changed often, what do you think?
4: i believe the failure was non-determinism due to a lack of an `ORDER BY` clause, ive rewritten the tests to use a `Set` when doing `==`, but that needs an `Ord TxOut`, so the real query used by explorer is still a `[a]`, not a `Set a`

File: cardano-explorer-db/cardano-explorer-db.cabal (hunk 1 above)
Comment (cleverca22) [reply cardano-explorer-db/cardano-explorer-db.cabal]: i'll get that tomorrow, its getting late here

**Review by erikd — DISMISSED** [review]

**Pull request merged**